### DISCUSSION TITLE
Multivariate flag behavior when changing weightings

### DISCUSSION THREAD

**Discussion by solugebefola** [Flagsmith/flagsmith#2836] [Q&A] [answered]
Hi there,

I was wondering what the behavior is for multivariate flags when weightings change?  Do already established identities change, or only new ones?
Ex.  If there is a multivariate flag with two values "control" and "experiment" at 50/50 weighting, and the weighting is changed to 100/0, will all existing identities be changed to "control"?

**Comment by dabeeeenster** [comment, marked as answer]
Detailed here: https://docs.flagsmith.com/guides-and-examples/staged-feature-rollouts#how-does-it-work

So yes, 100/0 everyone will get control. If you are in the 30% split, and it is increase to 31%, you are guaranteed to remain in that group. If you reduce it from 30% to 29%, you are not guaranteed to be in that group.
